examples/static-site/Caddyfile

```diff
@@ -0,0 +1,5 @@
+0.0.0.0:8080 {
+        root /var/www/public
+        log stdout
+        gzip
```

Comment:
Thanks, @rporres by noticing it!
@erizocosmico @Serabe I think it's a good idea to include it in the template because that way all projects will be benefited by this performance improvement.